### ISSUE TITLE
Railroads eaten by nearby parallel roads

### ISSUE THREAD

**Issue by jidanni** [der-stefan/OpenTopoMap#331]
![Screenshot_20221108-210813_All-In-One Offline Maps.jpg](https://user-images.githubusercontent.com/1230959/200728831-25dd59a5-7a3a-45f2-8306-803beec15c8b.jpg)
It's not fair. What if instead roads got eaten by nearby railroads?

**Comment by der-stefan** [comment]
Yes, that is true. As Mapnik does not have a generalization algorithm for shifting two neighoring lines apart. So which line should win? We decided that roads should be rendered on top, as navigation is more important for car drivers/cyclists etc. than for locomotive drivers.

**Comment by jidanni** [comment]
Hey we're taking green public transportation: the Amtrak Texas Eagle train! But I guess those old gas guzzlers  cars are more important... sigh.
